### PR TITLE
fix: design fixes for keystatistics component

### PR DESCRIPTION
**Pull request by dcshzj**
Closes ISOM-966.

Design to confirm:
- [ ] How to handle large numbers as one of the key statistics item

### CODE REVIEW

**Comment by dcshzj** [comment]
!run chromatic

File: src/templates/next/components/KeyStatistics/KeyStatistics.tsx [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+  Record<number, string>
+> = {
+  side: {
+    1: "basis-full",
+    2: "basis-[calc((100%-2.5rem)/2)]",
+    3: "basis-[calc((100%-5rem)/3)]",
+  },
+  top: {
+    1: "basis-full",
+    2: "basis-[calc((100%-2.5rem)/2)]",
```
Comment (harishv7) [inline src/templates/next/components/KeyStatistics/KeyStatistics.tsx:15]: do we need to do these calc ourselves or can consider using presets?

**Review by harishv7 — APPROVED** [review]
lgtm, minor comment

File: src/templates/next/components/KeyStatistics/KeyStatistics.tsx (hunk 1 above)
Comment (dcshzj) [reply src/templates/next/components/KeyStatistics/KeyStatistics.tsx]: So far this is something that is quite specific for key statistics component, so opted to not use presets for this.

**Pull request merged**